### PR TITLE
Feat: Add mainnet projects

### LINKED ISSUE #13: Implement Metadata for each Mainnet project
**Description**
---
Implement Metadata for each Mainnet project
- MANJARISOA 
- BANEGAS FARMS
- LAS DELICIAS

**Acceptance Criteria**
---

- [ ] Implement Metadata for MANJARISOA 
- [ ]  Implement Metadata for BANEGAS FARMS
- [ ]   Implement Metadata for LAS DELICIAS

### PR DESCRIPTION
**Pull request by tekkac**
closes #13

### CODE REVIEW

File: src/lib.cairo [hunk 1]
```diff
@@ -38,6 +39,9 @@ mod components {
         }
         mod jpegs {
             mod farmer;
+            mod parrot;
+            mod swamp;
+            mod monkey;
```
Comment (bal7hazar) [inline src/lib.cairo:44]: What about if another project is a different monkey? Maybe we should keep the project names.
Or be more precise on the animal type?

File: src/interfaces/erc165.cairo [hunk 2]
```diff
@@ -1,50 +1,4 @@
...
-            assert(interface_id != super::INVALID_ID, 'Invalid id');
-            self.supported_interfaces.write(interface_id, true);
-        }
-
-        fn deregister_interface(ref self: ContractState, interface_id: u32) {
-            assert(interface_id != super::IERC165_ID, 'Invalid id');
-            self.supported_interfaces.write(interface_id, false);
-        }
-    }
+trait IERC165<T> {
```
Comment (bal7hazar) [inline src/interfaces/erc165.cairo:2]: Why `TContractState` -> `T`? `TContractState` has the advantage to provide some guidance for dev.

**Review by bal7hazar — APPROVED** [review]
None blocking points

File: src/lib.cairo (hunk 1 above)
Comment (tekkac) [reply src/lib.cairo:44]: Noted, might change in another PR

File: src/interfaces/erc165.cairo (hunk 2 above)
Comment (tekkac) [reply src/interfaces/erc165.cairo:2]: `T` might change in the future, especially in this particular case where with components
cf [this code](https://github.com/starkware-libs/cairo/blob/b948fd96f5a16a29d48001ca33f21fd499a87fbc/crates/cairo-lang-starknet/test_data/test_contract.cairo#L4)

**Pull request merged**